lib/sdf-server/src/service/v2/variant/list_variants.rs

```diff
@@ -39,21 +38,13 @@ pub async fn list_variants(
         ));
     }
 
-    let mut user_scoped_modules: Vec<CachedModule> = vec![];
```

Comment:
This is the place we were using latest_modules and concatenating with user-scoped modules.